### ISSUE TITLE
Longshot

### ISSUE THREAD

**Issue by jcox10** [jbasilico/Test#2]
Hi @jbasilico I know this is odd and a total longshot. I got your name from the SNL cognitive foundry project. I am trying to get access to the Maven Central namespace for gov.sandia, and I wonder if you were the one that originally set it up? Cognitive foundry was the only project to every publish there. Hopefully this reaches you!

**Comment by jbasilico** [comment]
Yeah sure.On Oct 11, 2024, at 9:26 AM, Jason Cox ***@***.***> wrote:﻿
Hi @jbasilico I know this is odd and a total longshot. I got your name from the SNL cognitive foundry project. I am trying to get access to the Maven Central namespace for gov.sandia, and I wonder if you were the one that originally set it up? Cognitive foundry was the only project to every publish there. Hopefully this reaches you!

**Comment by jcox10** [comment]
thanks! maybe just email me at jascox@sandia.gov and I can tell you more.
